crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -311,8 +311,7 @@ where
     /// If automatically inserting [`apply_deferred`](crate::schedule::apply_deferred) like
     /// this isn't desired, use [`before_ignore_deferred`](Self::before_ignore_deferred) instead.
     ///
-    /// Note: The given set is not implicitly added to the schedule when this system set is added.
-    /// It is safe, but no dependencies will be created.
+    /// Note: please check the [Notes of `after`](Self::after)
```

Comment:
```suggestion
    /// Note: please check the [notes of `after`](Self::after) before you use this method.
```
Bit more cautioning is good here imo

Reply:
Your suggestion is certainly better and includes the change from Note to Caveat. But what do you mean by cautioning?